server/src/core/server/services/users/emailAliasBanFilter.ts

```diff
@@ -0,0 +1,36 @@
+import { MongoContext } from "coral-server/data/context";
+import { parseEmailAliasIntoParts, sanitizeStringForRegex } from "./helpers";
+
+export const shouldBanEmailBecauseOtherAliasesAreBanned = async (
```

Comment:
I think we need to add a check for premoderate email aliases being enabled in here. I tested out creating a new email alias when another was banned, and with the feature disabled, it still banned the new user.

Reply:
fixed

Reply:
hmm I am still seeing my new email alias user being banned by the system when the feature is disabled.

Reply:
I think `tooManyPeriods` is being checked where `emailAliases` should be.

Reply:
yeah, that's definitely it, fixed!